src/assets/scss/components/_featuresComponent.scss

```diff
@@ -1,17 +1,21 @@
-.features ul{
-	position: relative;
-	top: 20px;
-	left: 40px;
-}
+// .features ul{
```

Comment:
You can delete stuff instead, git provides good history in case you need to roll-back